observability/observability.go

```diff
@@ -88,31 +88,31 @@ var AllViews = []*view.View{
 	ViewExporterDroppedSpans,
 }
 
-// ContextWithReceiverName adds the tag "oc_receiver" and the name of the receiver as the value,
+// ContextWithReceiverName adds the tag "otelsvc_receiver" and the name of the receiver as the value,
 // and returns the newly created context. For receivers that can receive multiple signals it is
 // recommended to encode the signal as suffix (e.g. "oc_trace" and "oc_metrics").
 func ContextWithReceiverName(ctx context.Context, receiverName string) context.Context {
-	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyReceiver, receiverName))
+	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyReceiver, receiverName, tag.WithTTL(tag.TTLNoPropagation)))
 	return ctx
 }
 
-// RecordTraceReceiverMetrics records the number of the spans received and dropped by the receiver.
+// RecordMetricsForTraceReceiver records the number of the spans received and dropped by the receiver.
 // Use it with a context.Context generated using ContextWithReceiverName().
-func RecordTraceReceiverMetrics(ctxWithTraceReceiverName context.Context, receivedSpans int, droppedSpans int) {
+func RecordMetricsForTraceReceiver(ctxWithTraceReceiverName context.Context, receivedSpans int, droppedSpans int) {
 	stats.Record(ctxWithTraceReceiverName, mReceiverReceivedSpans.M(int64(receivedSpans)), mReceiverDroppedSpans.M(int64(droppedSpans)))
 }
 
-// ContextWithExporterName adds the tag "oc_exporter" and the name of the exporter as the value,
+// ContextWithExporterName adds the tag "otelsvc_exporter" and the name of the exporter as the value,
 // and returns the newly created context. For exporters that can export multiple signals it is
 // recommended to encode the signal as suffix (e.g. "oc_trace" and "oc_metrics").
 func ContextWithExporterName(ctx context.Context, exporterName string) context.Context {
-	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyExporter, exporterName))
+	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyExporter, exporterName, tag.WithTTL(tag.TTLNoPropagation)))
```

Comment:
@bogdandrutu I am not familiar with how tags work. Can you please clarify why this change is needed?

Reply:
Updated description. Also this just limits the scope of the tags to the current binary and does not propagate them to the next hope.